packages/framework-integration-tests/package.json

```diff
@@ -48,7 +48,7 @@
     "fix-lint": "eslint --quiet --fix --ext '.js,.ts' **/*.ts",
     "compile": "tsc -b tsconfig.json",
     "clean": "rimraf ./dist tsconfig.tsbuildinfo",
-    "integration": "FULL_INTEGRATION_TEST=true yarn integration/aws && yarn integration/local && yarn integration/cli && yarn integration/end-to-end",
+    "integration": "FULL_INTEGRATION_TEST=true yarn integration/local && yarn integration/cli && yarn integration/aws && yarn integration/end-to-end",
```

Comment:
It's better to make the tests that run locally to fail earlier. We were passing the whole AWS suite just to hit the error in the CLI before, it sucked!